plugins/source/cloudflare/codegen/recipes/account.go

```diff
@@ -23,9 +23,9 @@ func AccountResources() []Resource {
 			CFStruct: &cloudflare.AccountMember{},
 			DefaultColumns: []codegen.ColumnDefinition{
 				{
-					Name:     "account_cq_id",
-					Type:     schema.TypeUUID,
-					Resolver: "schema.ParentIDResolver",
+					Name:     "account_id",
```

Comment:
This was updated in the resources, but not in the code gen (I was the one who did this update and missed the code gen changes)